acceptance/acceptance_test.go

```diff
@@ -197,6 +205,24 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		tmpDir = t.TempDir()
 	}
 
+	cmdEnv := os.Environ()
+
+	// Start a new server with a custom configuration if the acceptance test
+	// specifies a custom server stubs.
+	if len(config.Server) > 0 {
+		server := testserver.New(t)
+
+		for _, stub := range config.Server {
+			require.NotEmpty(t, stub.Pattern)
+			require.NotEmpty(t, stub.Response.Body)
```

Comment:
Why require this non-empty body? It's up to users.

Reply:
Because we explicitly send a request JSON body below. Use cases where the API does not send a response body are very rare (Only one API comes to mind off the top of my head). We can handle those when we get there.

Reply:
Let's not limit config unnecessarily. If someone puts empty string there, that's what they should receive. I might want to check what happens on broken responses.

Reply:
removed this validation.